### PR TITLE
M365 CLI - Apply column format to SharePoint column

### LINKED ISSUE #272: CLI for M365 version for script "Apply column format to SharePoint column"
🎯Aim
The aim of this issue would be to add a [CLI for M365](https://pnp.github.io/cli-microsoft365/) version of script [Apply column format to SharePoint column](https://pnp.github.io/script-samples/spo-apply-column-format/README.html?tabs=pnpps).

🤔How
I think one of the best approach would be to use [field-set](https://pnp.github.io/cli-microsoft365/cmd/spo/field/field-set/) and [field-get](https://pnp.github.io/cli-microsoft365/cmd/spo/field/field-get/) command

### PR DESCRIPTION
**Pull request by Jwaegebaert**
Closes #272

### CODE REVIEW

File: scripts/spo-apply-column-format/README.md [hunk 1]
```diff
@@ -90,6 +89,77 @@ $fieldToFormat.UpdateAndPushChanges($true)
...
+        "class": "ms-fontColor-neutralSecondary"
+      }
+    }
+  ]
+}
+'@
+
+m365 spo field set --webUrl $url --listTitle $listName --name "ChoiceFldFormattedViaPowershell" --CustomFormatter $fieldFormat.Replace('"', '""')
+
+#----method 3 Apply formattin using sample from github-------#
```
Comment (Adam-it) [inline scripts/spo-apply-column-format/README.md:154]: I suppose it should be `formatting` 🤔

File: scripts/spo-apply-column-format/README.md [hunk 2]
```diff
@@ -90,6 +89,77 @@ $fieldToFormat.UpdateAndPushChanges($true)
 > [!Note]
 > In 3rd method i use **$fieldToFormat.UpdateAndPushChanges($true)** but it is perfectly fine to use **Set-PnPField** command as in method 1 and 2
 
+# [CLI for Microsoft 365](#tab/cli-m365-ps)
+```powershell
+#site collection url
+$url="https://thejasey.sharepoint.com/sites/Examplesite"
+
+#list to be exported  
+$listName="Test"
```
Comment (Adam-it) [inline scripts/spo-apply-column-format/README.md:98]: lets just add space next to `=` to increase readability and make it consistent with rest of the code 👍
`$url = "https://thejasey.sharepoint.com/sites/Examplesite"`
and
`$listName = "Test"`

**Review by Adam-it — COMMENTED** [review]
pulled locally and tested ✅
the second and third method in my case produced quite strange field formating 
image
but I guess it's not that important as here we just want to show how to use CLI not list formatting so for me its ok 🙂

thanks for your great work 🤩 
Please consider my small comments to make it even better 👍
your rock 🤩

**Review by Adam-it — APPROVED** [review]
Thanks @Jwaegebaert for your amazing work. You rock 🤩.
@pkbullock I guess it's now waiting for you second opinion 👍

**Comment by pkbullock** [comment]
If it's ok I'll process at the end of the week, I am on holiday without my computer - not all the processes I use for samples are automated 🤦🏻‍♂️

**Comment by Adam-it** [comment]
@pkbullock sure thing 👍. Have a great holiday 👍👍

**Comment by pkbullock** [comment]
Confirmed this works with the generator - thank you @Jwaegebaert another great sample!! 🎉👏

**Pull request merged**